TundraTD/Assets/Scripts/Camera/CameraMotion.cs

```diff
@@ -0,0 +1,201 @@
+﻿using System;
+using UnityEngine;
+
+// У нас по-настоящему инклюзивная, гендерно нейтральная команда!
+public class CameraMotion : MonoBehaviour
+{
+    private Camera Cam;
+    private bool Flag;
```

Comment:
Use more descriptive variable names. Don't use short variable names. Same for ChangePos1 & 2, xPos & zPos